make-based/include/common_command

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 
-if test $# -ne 1; then
+usage()
+{
     echo "Usage: $0 <command>" 1>&2
     echo "  command: setup configure build clean run run_debug remove" 1>&2
     exit 1
```

Comment:
Do not place `exit 1` in `usage()` function. Leave it outside.

Reply:
Makes sense, fixed!